src/Components/Game.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './Game.css'
+export default class Game extends React.Component {
+    state = {
+        score: 0,
+        points: 1
+    }
+    handleStart = () => {
+        this.setState({
+            score: 0,
+            points: 1
+        })
+    }
+
+    handlePoints = () => {
+        this.setState({score: this.state.score + this.state.points})
+    }
+
+    handleBonus = () => {
+        if (this.state.score > 10) {
+            this.setState({
+                points: this.state.points + 1,
+                score: this.state.score - 10
+            })
+        } else {
+            window.alert("You don't have enough points")
```

Comment:
Minimize the usage of using alert, this can become annoying to the user, so its better to create a div that display what youre trying to get across